.gitignore

```diff
@@ -0,0 +1,19 @@
+ros2_ws/build
+ros2_ws/install
+ros2_ws/log
+packaging/*.deb
+
+# needed for CLion compatibility
+.idea
+CMakeLists.txt
```

Comment:
Should this file be kept visible to git changes?

Reply:
there is no CMakeLists.txt in the root repo, and also `colcon build` don't need it.
I locally have my own CMakeLists.txt for integration with CLion IDE.
So this line just for myself :) I don't want accidentally push it

Reply:
Ok. Clear.